tests/base/assert.cpp

```diff
@@ -31,44 +31,41 @@ TEST_CASE("Test assert", "[base]")
 
 	// Best way to test different behavior of emp_assert based on compiler flags?
 
+  int expected_line_num = __LINE__ + 1; // This line must precede the assert for line num to pass
 	emp_assert(false);
 	REQUIRE(emp::assert_last_fail);
 
-	/* not sure why file is assert.h and line_num is 95, is this expected behavior?
+  // Test filename and line number
+  // Trim the filename in case it runs from another directory
 	std::string filen = emp::assert_fail_info.filename;
```

Comment:
Const and we should just use the full word filename — we’re not paying by the character